django_geosource/models.py

```diff
@@ -185,9 +200,15 @@ def update_fields(self):
                         try:
                             value = value.decode()
                         except (UnicodeDecodeError, AttributeError):
-                            logger.warning(
+                            msg = (
                                 f"{field_name} couldn't be decoded for source {self.pk}"
```

Comment:
Pk is not really human readable. Use name (beware of encoding, test with accentuated source name)

Reply:
Done